macos/FluentUI/Button/Button.swift

```diff
@@ -64,7 +64,7 @@ open class Button: NSButton {
 
 	/// Initializes a Fluent UI Button with no title or image, and with default formatting
 	@objc public convenience init() {
-		self.init()
+		self.init(title: nil, image: nil, imagePosition: .imageLeading, format: ButtonFormat())
```

Comment:
Maybe set title to "" instead of nil here? Then you wouldn't need the default override below?

Reply:
`nil` makes an important distinction between a "button with a title", a "button with an image", and "a button with both a title and an image".